exercises/nth-prime/.meta/template.j2

```diff
@@ -0,0 +1,36 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if not case["expected"]["error"] is string -%}
```

Comment:
There's a built-in filter you can use here:
```suggestion
        {% if not case is error_case -%}
```

For readability, I would suggest switching your cases around so you can just use `{% if case is error_case -%}`

Reply:
wow, that is a much better way to do that! I totally missed that generate_test.py was adding a filter for that. Thanks for pointing that out!

Reply:
Documenting which non-standard filters are built in to the generator is somewhere on my (very long) TODO list.